### PR TITLE
chore!: bump dependency send@3

### PR DESCRIPTION
**Pull request by climba03003**
Note that the current test fail is due to @fastify/compress
https://github.com/fastify/fastify-compress/blob/14b83fe235000ed2620d2387b02f84b4170f55fa/index.js#L568-L573

Depends on https://github.com/fastify/send/pull/80
We should wait until the next release of both `@fastify/send` and `@fastify/compress`.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test/dir-list.test.js [hunk 1]
```diff
@@ -471,7 +471,7 @@ t.test('dir list json format - extended info', t => {
           t.equal(response.statusCode, 200)
           const bodyObject = JSON.parse(body.toString())
           t.equal(bodyObject.dirs[0].name, 'empty')
-          t.equal(typeof bodyObject.dirs[0].stats.atime, 'string')
+          t.equal(typeof bodyObject.dirs[0].stats.atimeMs, 'number')
```
Comment (climba03003) [inline test/dir-list.test.js:474]: I believe there is bug on `Node.js` side, `JSON.stringify` do not print `atime` directly.

**Review by mcollina — APPROVED** [review]
lgtm once CI is green

**Comment by mcollina** [comment]
Can you update the actions too? We don't support Node v18 anymore.

**Comment by climba03003** [comment]
Depends on https://github.com/fastify/fastify-compress/pull/300

**Review by jsumners — APPROVED** [review]
Thank you.

**Pull request merged**